### DISCUSSION TITLE
23 gennaio 2023  A - esercizio 1

### DISCUSSION THREAD

**Discussion by atyion** [sapienzastudentsnetwork/basi-di-dati-1#20] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/cfdf3cc7-e41c-4725-a015-7e57a4906ec9)

**Comment by niccolo-fato** [comment]
![2024-01-13 16 19 24](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/4191e986-19c8-45fa-aeed-cbe6ace01d31)

**Reply by MattiaGio** [reply]
anche io ho fatto così

**Comment by atyion** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/d34d63ea-0bf2-451b-8411-a9ce57355b3a)

**Reply by niccolo-fato** [reply]
Nel 2b teoricamente devi prendere il personale che fa partenze diverse da Roma per poi sottrarle al personale, nel tuo caso stai prendendo gli arrivi se non mi sbaglio

**Reply by atyion** [reply]
yup, correggo. Oggi sono in vena di errori.

**Comment by Elia-Belli** [comment]
**(1)**
- Voli per Berlino: $Voli\\_ BE=(Volo\bowtie_{(Arrivo=BE)} (\rho_{(BE\leftarrow Sigla)}(\sigma_{(Città= Berlino)}Aeroporto)))$
- Viaggi per Berlino nel 2022: $Viaggi\\_ BE\bowtie_{(Sigla= SiglaVolo)}(\sigma_{(01-01-2019\leq Data\lt 01-01-2020)}Viaggio)$
- Aerei revisionati prima di 5 anni dalla costruzione: $Aerei= \sigma_{(AnnoR-AnnoC\lt 5)}Aereo$
- Risposta: $\pi_{(Modello)}(Aerei\bowtie_{(ID=Aereo)}(\rho_{(VID\leftarrow ID)}Viaggi\\_ BE))$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/79638284/80dd7386-00b6-4ef2-9f8e-d9ea83ba9118)
